bundles/af-core/src/test/resources/form/textinput/exporter-textinput-customized.json

```diff
@@ -4,8 +4,8 @@
   "fieldType": "text-input",
   "name": "abc",
   "visible": false,
-  "description": "dummy",
-  "tooltip": "test-short-description",
+  "description": "&lt;p&gt;dummy&lt;/p&gt;",
```

Comment:
This would need to be part of crispr spec as wordings and we would have to bump major version of adaptive form property which we emit today, so that existing clients do not break in headless

Reply:
Avoiding this route